igvm/hypervisor.py

```diff
@@ -567,37 +566,45 @@ def migrate_vm(
                 .format(offline_transport)
             )
         disk_size = disk_size if 0 < disk_size else 0
-        if offline_transport != 'xfs' and 0 < disk_size:
+        if not (offline and offline_transport == 'xfs') and 0 < disk_size:
```

Comment:
I'm trying to find an article where I've read about the positive side of using `<` always.

If shortly, because it uses the "natural" x axis `------->`, it allows decreasing the brain load to parse the conditions.

I'm doing my best to find the article. =\

Reply:
Here I've found a hint why is it better https://flylib.com/books/en/2.823.1.155/1/, Search on the page by `Writing Numeric Expressions in Number-Line Order`

Reply:
I agree with @TuxPowered42. 

Even if your article has an interesting point in here we should stick to what we use more often in the whole project and not change the style in at some places in the project.

Please take this into account for further comparisons too.

Reply:
More verbose description http://llewellynfalco.blogspot.com/2016/02/dont-use-greater-than-sign-in.html